bin/collator/src/evm_tracing_types.rs

```diff
@@ -50,7 +51,7 @@ impl std::str::FromStr for EthApi {
 #[allow(dead_code)]
 #[derive(Clone)]
 /// EVM tracing CLI config.
-pub struct EvmTracingConfig {
+pub struct RpcConfig {
```

Comment:
I guess this is fine but you should also update the doc above and the variable shouldn't be called `evm_tracing_config` anymore (check the usage).

Alternative is to have a struct with two embedded configs - one for tracing, one for the backend.

Reply:
I think expanding the scope of this trait is fine. As it's mainly for EVM frontier config

Reply:
I'm not against it, it's perfectly fine.

Maybe the name/purpose is a bit misleading since I wouldn't call backend part of the RPC config.

Reply:
Updated to `FrontierConfig`